subsys/bluetooth/host/conn_internal.h

```diff
@@ -135,7 +135,10 @@ struct bt_conn {
 #if defined(CONFIG_BT_BREDR)
 		struct bt_conn_br	br;
 		struct bt_conn_sco	sco;
-#endif
+#endif /* CONFIG_BT_BREDR */
+#if defined(CONFIG_BT_TESTING)
+		struct bt_conn_rv_info rv;
```

Comment:
The union is for things which are mutually exclusive. Specifically, the idea is that the member used matches `conn->type`. It is therefore the wrong place to put bt_conn_rv_info.

Reply:
I actually fixed this like two weeks ago after Vudentz mentioned it, but I forgot to commit the change. It has now been pushed.